website/docs/patterns/filter-patterns/partials/specifications/specifications.md

```diff
@@ -0,0 +1,139 @@
+!!! Info
+
+Helios does not currently publish any components that account for layout of the structural filter elements, these are currently the responsibility of the consumer. Instead, use these guidelines in a manner that best supports your application structure.
+
+!!!
+
+## Pattern layout
+
+Adhering to the spacing, layout, and specifications outlined for the individual structural elements of a filter pattern makes assembling them together into a pattern relatively simple.
+
+**Ex 1.0: Pattern anatomy with a filter bar**
+
+![Pattern anatomy with filter bar](/assets/patterns/filter-patterns/pattern-anatomy-specific-filter-bar.png)
+
+**Ex 1.1: Pattern anatomy with a filter sidebar**
+
+![Pattern anatomy with a sidebar](/assets/patterns/filter-patterns/pattern-anatomy-specific-sidebar.png)
+
+| Element | Usage |
+|---------|-------|
+| Filters | Dictates what parameters are available to filter upon and the values that correspond with them. Can either be positioned in a horizontal filter bar or a vertical filter sidebar. |
+| Applied filters | Displays the filters that are applied to the dataset and accounts for the bulk clearing of filters. |
```

Comment:
Should we mention somewhere (if not already done) that these are optional? Otherwise consumers may infer that they _always_ have to add them (since most of the screenshots show them) which is not always the case.

Reply:
In general here I refrained from marking almost everything as either optional or required. I think what I was trying to do was imply that the decision on usage comes down to the specific application and the filtering strategy/implementation. I think I can be a bit more explicit in some cases, but I hesitate to be _too_ prescriptive.